app/src/main/java/com/example/todoapp/view/adapter/TaskAdapter.kt

```diff
@@ -0,0 +1,37 @@
+package com.example.todoapp.view.adapter
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.example.todoapp.R
+import com.example.todoapp.databinding.TodoItemBinding
+import com.example.todoapp.model.Task
+
+class TaskAdapter(
+    var list: ArrayList<Task>
+): RecyclerView.Adapter<TaskAdapter.TaskViewHolder>() {
+
+    class TaskViewHolder(view: View): RecyclerView.ViewHolder(view) {
```

Comment:
chỗ này em nên truyền vào TodoItemBinding thì scope hẹp hơn, lỗi trong runtime ít hơn 
```suggestion
    class TaskViewHolder(val binding: TodoItemBinding): RecyclerView.ViewHolder(binding.root) {
```